<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_Teacher</class>
 <widget class="QMainWindow" name="MainWindow_Teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生成绩管理系统-教师模式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/image/images/背景.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_fixed">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>478</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>欢迎使用学生成绩管理系统！尊敬的老师:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>100000</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_teacher_total">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>教职工号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>任教课数</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_teacher">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>课程名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>课程号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>学分</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>修课人数</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_lesson_total">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>401</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>课程名</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>课程号</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>任课教师</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>任课教师号</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>学分</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>修课人数</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>平均成绩</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>平均绩点</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>中位成绩</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>中位绩点</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>通过率</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>挂科率</string>
     </property>
    </row>
   </widget>
   <widget class="QTableWidget" name="table_lesson">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>学号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成绩</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>绩点</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>等级</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_back"/>
   <addaction name="action_relog"/>
   <addaction name="action_leave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_stuid"/>
   <addaction name="action_score"/>
   <addaction name="action_setsort"/>
   <addaction name="separator"/>
   <addaction name="action_look"/>
   <addaction name="action_edit"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_look">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/查看.png</normaloff>:/image/images/查看.png</iconset>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </action>
  <action name="action_setsort">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/排序.png</normaloff>:/image/images/排序.png</iconset>
   </property>
   <property name="text">
    <string>设置升序/降序</string>
   </property>
  </action>
  <action name="action_score">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/成绩.png</normaloff>:/image/images/成绩.png</iconset>
   </property>
   <property name="text">
    <string>成绩</string>
   </property>
  </action>
  <action name="action_stuid">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/学号.png</normaloff>:/image/images/学号.png</iconset>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/编辑.png</normaloff>:/image/images/编辑.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/保存.png</normaloff>:/image/images/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_back">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/后退.png</normaloff>:/image/images/后退.png</iconset>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </action>
  <action name="action_relog">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/注销.png</normaloff>:/image/images/注销.png</iconset>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="action_leave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/image/images/退出.png</normaloff>:/image/images/退出.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
